docs/general/emulator-support-and-issues.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Emulator Support
-description: Comprehensive guide to emulator support and compatibility for RetroAchievements, including console-specific issues and recommendations for various cores and emulators.
+Title: Emulator Support
+Description: Comprehensive guide to emulator support and compatibility for RetroAchievements, including console-specific issues and recommendations for various cores and emulators.
```

Comment:
This is not [conventional frontmatter syntax](https://vitepress.dev/guide/frontmatter). Please revert.